src/components/global-header/global-header.pw.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import {
+  FullMenuExample,
+  GlobalHeaderWithErrorHandler,
+  GlobalHeaderWithLogo,
+} from "./component.test-pw";
+import navigationBar from "../../../playwright/components/navigation-bar";
+import {
+  globalHeader,
+  globalHeaderLogo,
+} from "../../../playwright/components/global-header";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+test.describe("Global Header component", () => {
+  test("should not cause a ResizeObserver-related error to occur", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<GlobalHeaderWithErrorHandler />);
+
+    await page.waitForTimeout(500);
+
+    const errorState = page.locator("#error-div");
+    await expect(errorState).toHaveText("");
+  });
+
+  test("should check that z-index of component is greater than that of NavigationBar", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<FullMenuExample />);
+
+    const globalHeaderZIndex = await globalHeader(page).evaluate((element) => {
+      const style = getComputedStyle(element);
+      return style.zIndex;
+    });
+
+    const navigationBarZIndex = await navigationBar(page).evaluate(
+      (element) => {
+        const style = getComputedStyle(element);
+        return style.zIndex;
+      }
+    );
+
+    const globalIndex = parseInt(globalHeaderZIndex.toString());
+    const NavIndex = parseInt(navigationBarZIndex.toString());
+
+    expect(globalIndex).toBeGreaterThan(NavIndex);
+  });
+
+  test("should check when logo prop is passed, the height of the logo element never exceeds the maximum height of the component", async ({
+    mount,
+    page,
+  }) => {
+    const expectedHeight = 40;
+
+    await mount(<GlobalHeaderWithLogo />);
+
+    await expect(globalHeaderLogo(page)).toHaveCSS(
+      "height",
+      `${expectedHeight}px`
+    );
```

Comment:
So if you implement my previous comment, this should work. 

```suggestion
    await expect(globalHeaderLogo(page)).toHaveCSS(
      "height",
      `40px`
    );
```